lua/noice/lsp/message.lua

```diff
@@ -17,7 +17,7 @@ M.message_type = {
 ---@alias ShowMessageParams {type:MessageType, message:string}
 
 function M.setup()
-  vim.lsp.handlers["window/showMessage"] = M._on_message
+  vim.lsp.handlers["window/showMessage"] = M.on_message
```

Comment:
I was looking though git blame. You might want to use `Util.protect(M.on_message)` to restore the previous behavior. 

See [0f5f8c93](https://github.com/folke/noice.nvim/commit/0f5f8c93dd11546574640c559f5843488e460df6#diff-260044f29fc9da56e284356e8acd69f15dba10a8088d59d41de62477567e0bc5L20)
```suggestion
  vim.lsp.handlers["window/showMessage"] = Util.protect(M.on_message)
```